crabpy_pyramid/views/adressenregister.py

```diff
@@ -177,7 +177,10 @@ def list_straten(request):
     request = set_http_caching(request, "adressenregister", "short")
     Gateway = request.adressenregister_gateway()
     gemeente_niscode = request.matchdict.get("niscode")
-    straten = handle_gateway_response(Gateway.list_straten, gemeente_niscode)
+    include_homoniem = True if request.params.get("include_homoniem") else False
```

Comment:
deze change moet ook nog in `get_straat_by_id`

Reply:
Daar maakt die denk ik niet uit? Omdat de gateway method daar zelf ook niks mee doet?